remindme-api/src/main/java/it/andreafailli/remindme/RemindMeApiApplication.java

```diff
@@ -58,5 +48,22 @@ public Docket swaggerConfiguration() {
     	        .build()))
           .securitySchemes(Lists.newArrayList(new ApiKey(FirebaseAuthFilter.HEADER_NAME, FirebaseAuthFilter.HEADER_NAME, ApiKeyVehicle.HEADER.getValue())));
     }
+	
+	@Bean
+	public FirebaseAuth firebaseAuth() {
+		if (FirebaseApp.getApps().isEmpty()) {
+			try {
+				InputStream inputStream = RemindMeApiApplication.class.getClassLoader().getResourceAsStream("firebase-server-key.json");
+				FirebaseOptions options = new FirebaseOptions.Builder()
+					  .setCredentials(GoogleCredentials.fromStream(inputStream))
+					  .setDatabaseUrl("https://glass-crossing.firebaseio.com")
+					  .build();
+				FirebaseApp.initializeApp(options);
+			} catch (IOException e) {
+				throw new RuntimeException(e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define and throw a dedicated exception instead of using a generic one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00112)